### ISSUE TITLE
[BUG] pot stop does not check for dns=off before deleting /resolv.conf

### ISSUE THREAD

**Issue by paulc** [pizzamig/pot#117]
**Describe the bug**

Hi,

Pull request #114 added the `dns=off` option (thanks!) however unfortunately `pot stop` will still unilaterally delete `/etc/resolv.conf` when run (which will delete any local configuration). 

**To Reproduce**

Create pot with `pot create -d off ...`
Start pot and populate `/etc/resolv.conf`
Stop pot
Restart pot and notice that `/etc/resolv.conf` is gone

**Expected behavior**

`/etc/resolv.conf` should be retained if `dns=off`

The following patch checks for `dns=off` and skips deleting the file in this case.

```patch
diff --git a/share/pot/stop.sh b/share/pot/stop.sh
index eb3c5dc..e3f20f0 100644
--- a/share/pot/stop.sh
+++ b/share/pot/stop.sh
@@ -116,7 +116,8 @@ _js_rm_resolv()
        local _pname _jdir
        _pname="$1"
        _jdir="${POT_FS_ROOT}/jails/$_pname"
-       if [ -f $_jdir/m/etc/resolv.conf ]; then
+       _dns="$(_get_conf_var $_pname pot.dns)"
+       if [ ! $_dns = 'off' -a -f $_jdir/m/etc/resolv.conf ]; then
                rm -f $_jdir/m/etc/resolv.conf
        fi
 }
```

**Comment by pizzamig** [comment]
Yes, you are right, I completely forgot to fix the stop part of it. I'll fix it ASAP.

**Comment by pizzamig** [comment]
This bug have been resolved in release 0.11.6.
Thanks again for reporting and for the patch
